contracts/staking/ERC20TokenStakingManager.sol

```diff
@@ -88,16 +88,21 @@ contract ERC20TokenStakingManager is
      * @param stakeAmount The amount to be staked.
      * @param nodeID The node ID of the validator being registered.
      * @param registrationExpiry The Unix timestamp after which the reigistration is no longer valid on the P-Chain.
+     * @param delegationFeeRate The fee charged to delegators by the validator.
      * @param blsPublicKey The BLS public key of the validator.
      */
     function initializeValidatorRegistration(
         uint256 stakeAmount,
         bytes32 nodeID,
         uint64 registrationExpiry,
+        uint256 delegationFeeRate,
         bytes memory blsPublicKey
     ) external returns (bytes32 validationID) {
```

Comment:
The inconsistency wasn't introduced in this PR, but we've generally tried to be consistent in not using named returned values. This has been a minor item that has popped up in past audits. Could you switch to using non-named returns here?